src/config/graphical-session/wayland.md

```diff
@@ -32,18 +32,22 @@ Void Linux currently packages the following Wayland compositors:
 Both GNOME and KDE Plasma have EGLStreams backends for Wayland, which means they
 can use the proprietary NVIDIA drivers. Most other Wayland compositors require
 drivers that implement the GBM interface. The main driver for this purpose is
-provided by the `mesa-dri` package. The [graphics
+provided by the `mesa-dri` package. The [graphics drivers
 section](./graphics-drivers/index.md) has more details regarding setting up
 graphics in different systems.
 
 ### Native applications
 
-Qt5 based applications require the installation of the `qt5-wayland` package and
-setting the environment variable `QT_QPA_PLATFORM=wayland-egl` to enable their
-Wayland backend. Some KDE specific applications might also require the
-installation of the `kwayland` package. GTK+ applications should do it
-automatically, but setting `GDK_BACKEND=wayland` can force the Wayland backend
-(however, this can break applications such as Chromium).
+[Qt5](https://wayland.freedesktop.org/qt5.html) based applications require the
+installation of the `qt5-wayland` package and setting the environment variable
+`QT_QPA_PLATFORM=wayland-egl` to enable their Wayland backend. Some KDE specific
+applications might also require the installation of the `kwayland` package.
+[EFL](https://wayland.freedesktop.org/efl.html) based applications require
+setting the environment variable `ELM_DISPLAY=wl`, and can have issues without
+it. GTK+ applications should use the Wayland backend automatically.
```

Comment:
"\[GTK+\](https://gtk.org/)-based", to match style used above when referencing other tooklkits.

Reply:
I'm using links from the Wayland page about toolkits, so it might be good to link said page. Let me know what you think.

Reply:
i.e. https://wiki.gnome.org/Initiatives/Wayland/GTK%2B ? Sure; my concern here is that "GTK+" be a link, like the other toolkit mentions are. (And that there be a hyphen between \<toolkit name\> and "based".)

Reply:
I believe that's solved, then. Thanks!

Reply:
The `ELM_DISPLAY` var didn't seem to be needed at least when actually running Enlightenment but shrug

Reply:
I tried to make that clear in the `desktop environment` block. If you think it could be rephrased, I'd be glad to do so. The issue presents itself with Sway when XWayland is enabled. It's this bug: https://phab.enlightenment.org/T7439

Reply:
I guess it may still be problematic even in E when xwayland is running, but not sure... it shouldn't hurt to set it anyway, i guess